migrations/2018_07_22_000000_create_login_providers_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Flarum\Database\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+return Migration::createTable(
+    'login_providers',
+    function (Blueprint $table) {
+        $table->increments('id');
+        $table->unsignedInteger('user_id');
+        $table->string('provider', 100);
```

Comment:
Is provider supposed to be a FQCN? If so I'm not sure that 100 will suffice, some providers might have very long namespaces..

Reply:
Nah it's just a string to uniquely identify the provider. e.g. in 
https://github.com/flarum/flarum-ext-auth-facebook/pull/15 it's `facebook`.